src/Compilers/CSharp/Portable/Symbols/PropertySymbol.cs

```diff
@@ -321,6 +322,14 @@ internal virtual bool IsExplicitInterfaceImplementation
         /// </remarks>
         public abstract ImmutableArray<PropertySymbol> ExplicitInterfaceImplementations { get; }
 
+#nullable enable
+        internal virtual PropertySymbol? PartialImplementationPart => null;
+        internal virtual PropertySymbol? PartialDefinitionPart => null;
```

Comment:
nit: consider adjusting existing usages of `SourcePropertySymbol.PartialDefinitionPart`/`PartialImplementationPart` to be `prop.PartialDefinitionPart` instead of `(prop as SourcePropertySymbol)?.PartialDefinitionPart`